pyuploadcare/api/entities.py

```diff
@@ -80,15 +81,90 @@ class VideoInfo(Entity):
     duration: Decimal
     format: str
     bitrate: Decimal
-    audio: Optional[AudioStreamInfo]
-    video: VideoStreamInfo
+    audio: List[AudioStreamInfo]
+    video: List[VideoStreamInfo]
+
+
+class MIMEInfo(Entity):
+    mime: str
+    type: str
+    subtype: str
+
+
+class ContentInfo(Entity):
+    mime: Optional[MIMEInfo]
+    image: Optional[ImageInfo]
+    video: Optional[VideoInfo]
+
+
+class ApllicationDataDetails(Entity):
+    pass
+
+
+DetailsType = TypeVar("DetailsType", bound=ApllicationDataDetails)
+
+
+class ApplicationData(GenericModel, Generic[DetailsType]):
+    data: DetailsType
+    version: str
+    datetime_created: datetime
+    datetime_updated: datetime
+
+
+class AWSRecognitionLabel(Entity):
+    confidence: Decimal = Field(alias="Confidence")
+    instances: List[Dict] = Field(alias="Instances")
+    name: str = Field(alias="Name")
+    parents: List[Dict] = Field(alias="Parents")
+
+
+class AWSRecognitionDetectLabelsDetails(ApllicationDataDetails):
+    label_model_version: Optional[str] = Field(alias="LabelModelVersion")
+    labels: List[AWSRecognitionLabel] = Field(
+        alias="Labels", default_factory=list
+    )
+
+
+AWSRecognitionDetectLabelsApplicationData = ApplicationData[
+    AWSRecognitionDetectLabelsDetails
+]
+
+
+class RemoveBackgroundDetails(ApllicationDataDetails):
+    foreground_type: Optional[str]
+
+
+RemoveBackgroundApplicationData = ApplicationData[RemoveBackgroundDetails]
+
+
+class UCClamAVDetails(ApllicationDataDetails):
+    infected: Optional[bool]
+    infected_with: Optional[str]
+
+
+UCClamAVApplicationData = ApplicationData[UCClamAVDetails]
+
+
+class ApplicationDataSet(Entity):
+    aws_rekognition_detect_labels: Optional[
+        AWSRecognitionDetectLabelsApplicationData
+    ]
+    remove_bg: Optional[RemoveBackgroundApplicationData]
+    uc_clamav_virus_scan: Optional[UCClamAVApplicationData]
 
 
 class FileInfo(UUIDEntity):
+    """
+    video_info, image_info and rekognition_info were suppressed in API v0.7.0
```

Comment:
```suggestion
    video_info, image_info and rekognition_info were removed in API v0.7.0
```

Reply:
I agree that it is rather confusing usage of `suppressed`.
There is another example of a better word for substitution as Roman suggested in [PR216](https://github.com/uploadcare/pyuploadcare/commit/e36751b6dd2878d7d4afa1d19c52ccecd46bf5ed)
(so it should be resolved after final merging into release epic branch in [PR219](https://github.com/uploadcare/pyuploadcare/pull/219))

Reply:
bump